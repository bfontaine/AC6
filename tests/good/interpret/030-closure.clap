val x = 42
val f = fun (y) => x + y
val z = f 0

