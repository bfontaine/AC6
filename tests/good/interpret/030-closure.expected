:- x = 42
:- f = <code>
:- z = 42
